<?xml version="1.0"?>
<launch>
	<node pkg="avatar_face_recognition" type="face_finder.py" name="face_finder" output="screen" respawn="true">
		<param name="prototxt" value="$(dirname)/../deploy.prototxt" />
		<param name="weights" value="$(dirname)/../res10_300x300_ssd_iter_140000.caffemodel" />
		<param name="pickle" value="$(dirname)/../faces.pickle" />
		<param name="json" value="$(dirname)/../info.json" />
	</node>

	<node pkg="avatar_face_recognition" type="mask_detector.py" name="mask_detector" output="screen" respawn="true">
		<param name="mask_detector" value="$(dirname)/../mask_detector.model" />
	</node>

	<node pkg="avatar_face_recognition" type="face_2_screen.py" name="face_2_screen" output="screen"/>
</launch>
